.github/workflows/build.yaml

```diff
@@ -253,13 +253,15 @@ jobs:
         compiler:
           - clang
     name: MacOS ${{ matrix.config }} ${{ matrix.compiler }}
-    runs-on: macos-10.15  # Building with clang on MacOS 11 is currently broken
+    runs-on: macos-11
```

Comment:
Please also update the name of the artifact in the second-to-last line in this file with the changed version

Reply:
Done